docs/platforms/python/enriching-events/scopes/index.mdx

```diff
@@ -1,59 +1,42 @@
+instance, web framework integrations will create and destroy isolation scopes around your
+routes or controllers. When you call a global function such as <PlatformIdentifier name="capture-event" /> or <PlatformIdentifier name="set-tag" />, the SDK internally determines the correct scope to use to perform that functionality.
 
-As you start using an SDK, a scope and hub are automatically created for you out
-of the box. It's unlikely that you'll interact with the hub directly unless you're
-writing an integration or you want to create or destroy scopes. Scopes, on the
-other hand are more user facing. You can call <PlatformIdentifier name="configure-scope" /> at any point in
-time to modify data stored on the scope. This is useful for doing things like
-[modifying the context](../context/).
+At the end of the pipeline, the final data applied to the event is the result of merging the global scope and the isolation and current scopes together, applying data from each in turn.
 
-<Alert>
-If you are very curious about how thread locality works: On platforms such as .NET
-or on Python 3.7 and later we will use "ambient data" to have either the hub flow
-with your code or the hub is already a singleton that internally manages the scope.
+## Changing the Scope
 
-Effectively this means that when you spawn a task in .NET and the execution flow is
-not suppressed all the context you have bound to the scope in Sentry will flow along.
-If however you suppress the flow, you get new scope data.
+In most cases, it should be enough to use the top-level API (like <PlatformIdentifier name="capture-event" /> or <PlatformIdentifier name="set-tag" />) and let the SDK handle the scopes for you in the background.
 
-</Alert>
+If you do need to interact with scopes directly, you can use the <PlatformIdentifier name="new-scope" /> context manager. It can be used to fork the current scope, apply modifications to the new scope as long as the context manager is active, and then return to the original scope afterwards. This is useful if you only want to send data for one specific event, for instance for doing things like [modifying the context](../context/). It's roughly equivalent to the <PlatformIdentifier name="push-scope" /> context manager in earlier (1.x) versions of the SDK.
 
-When you call a global function such as <PlatformIdentifier name="capture-event" /> internally Sentry
-discovers the current hub and asks it to capture an event. Internally the hub will
-then merge the event with the topmost scope's data.
+<Note>
 
-## Configuring the Scope
+Make sure you're not mixing the use of <PlatformIdentifier name="new-scope" /> and top-level APIs -- this won't work well because the top-level API might interact with a different scope in the background. As long as you're in the <PlatformIdentifier name="new-scope" /> context manager, you should be calling methods directly on the scope you just created.
```

Comment:
This comment is somewhat vague. I tried to make it more specific and clearer, but I am unsure whether what I wrote is factually correct. If it is, let's apply this suggestion, otherwise please let me know what I misunderstood and we can come up with a better comment.

```suggestion
Avoid calling top-level APIs inside the  <PlatformIdentifier name="new-scope" /> context manager. The top-level API might interact with a different scope from what <PlatformIdentifier name="new-scope" /> yields, causing unintended results. While within the <PlatformIdentifier name="new-scope" /> context manager, please call methods directly on the scope that <PlatformIdentifier name="new-scope" /> yields!
```